Switch to "protobuf" module when generating proto_library rules (#2284)
bazelbuild/bazel-gazelle: README.md

* **Java:** bazel-contrib's [rules_jvm](https://github.com/bazel-contrib/rules_jvm) extensions include [a gazelle extension](https://github.com/bazel-contrib/rules_jvm/tree/main/java/gazelle) for generating `java_library`, `java_binary`, `java_test`, and `java_test_suite` rules.
* **JavaScript / TypeScript:** Aspect provides [JavaScript and TypeScript Support](https://github.com/aspect-build/aspect-gazelle/tree/main/language/js). BenchSci's [rules_nodejs_gazelle](https://github.com/benchsci/rules_nodejs_gazelle) supports generating `ts_project`, `js_library`, `jest_test`, and `web_asset` rules, and is able to support module bundlers like Webpack and Next.js.
* **Kotlin:** Aspect Build provides some [Kotlin Support](https://github.com/aspect-build/aspect-gazelle/tree/main/language/kotlin). Still under development, please check the README for currently available features.

> * **Protocol Buffers:** Support for the `proto_library` rule, as well as `go_proto_library` is in this repository, see below. Other language-specific proto rules are not supported here. [protocolbuffers/protobuf](https://github.com/protocolbuffers/protobuf/tree/main/bazel) is a good resource for these rules.

Reviewer: ```suggestion
* **Protocol Buffers:** Support for the `proto_library` rule. Language-specific rules like `go_proto_library` are supported by other language extensions.
```
